src/app/components/tfrs/tfrs.component.html

```diff
@@ -1 +1,92 @@
-<p>tfrs works!</p>
+<div>
+    <div>
+      <mat-form-field appearance="fill">
+        <mat-label>Vendor</mat-label>
+        <mat-select [(ngModel)]="selectedVendorName">
+          <mat-option>None</mat-option>
+          <mat-option *ngFor="let vendor of vendors" [value]="vendor.name">{{vendor.name}}</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Status</mat-label>
+        <mat-select [(ngModel)]="selectedStatus">
+          <mat-option>None</mat-option>
+          <mat-option *ngFor="let status of statusList" [value]="status">{{status}}</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project Start After</mat-label>
+        <input matInput [matDatepicker]="picker1" [formControl]="startAfterDate" >
+        <mat-hint>MM/DD/YYYY</mat-hint>
+        <mat-datepicker-toggle matIconSuffix [for]="picker1"></mat-datepicker-toggle>
+        <mat-datepicker #picker1></mat-datepicker>
+      </mat-form-field>
+
+      <mat-form-field appearance="fill">
+        <mat-label>Project End Before</mat-label>
+        <input matInput [matDatepicker]="picker2" [formControl]="endBeforeDate">
+        <mat-hint>MM/DD/YYYY</mat-hint>
```

Comment:
We should have this in constant file so if other also want to use this they can reuse this constant variable